rosie/core/__init__.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, settings, adapter):
         self.settings = settings
         self.dataset = adapter.dataset
         self.data_path = adapter.path
-        self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        if self.settings.UNIQUE_IDS:
+            self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        else:
+            self.suspicions = self.dataset.copy()
```

Comment:
Is there a chance we end up here without `UNIQUE_IDS`? I thought it was suppose to be a kind of standard/requirement…